KarhooUISDK/CommonUI/Views/KarhooTextInputView/KarhooPhoneInputView.swift

```diff
@@ -186,11 +187,17 @@ class KarhooPhoneInputView: UIView {
         let presenter = CountryCodeSelectionPresenter(preSelectedCountry: country) { [weak self] result in
             guard let value = result.completedValue()
             else {
+                if self?.focusPhoneNumber ?? false {
```

Comment:
Could you extract this `if` clause to a new method and call it here and in the success case after `self?.runValidation()`?